packages/console/src/pages/Connectors/index.tsx

```diff
@@ -42,30 +65,76 @@ const Connectors = () => {
               <td colSpan={2}>loading</td>
             </tr>
           )}
-          {data?.map(({ id, metadata: { name, logo }, enabled }) => (
-            <tr key={id}>
+          {!isLoading && (
+            <tr>
               <td>
-                <Link to={`/connectors/${id}`} className={styles.link}>
-                  <ItemPreview
-                    title={name[i18n.language] ?? name.en ?? '-'}
-                    subtitle={id}
-                    icon={
-                      logo.startsWith('http') ? (
-                        <img className={styles.logo} src={logo} />
-                      ) : (
-                        <ImagePlaceholder />
-                      )
-                    }
-                  />
-                </Link>
+                {emailConnector ? (
+                  <Link to={`/connectors/${emailConnector.id}`} className={styles.link}>
+                    <ItemPreview
+                      title={
+                        emailConnector.metadata.name[i18n.language] ??
+                        emailConnector.metadata.name.en ??
+                        '-'
```

Comment:
Won't i18n fall back to `en` by default?

Reply:
LOG-1845